util/find-doc-nits

```diff
@@ -577,7 +582,7 @@ if ( $opt_c ) {
     # See if each has a manpage.
     foreach ( @commands ) {
         next if $_ eq 'help' || $_ eq 'exit';
-        if ( ! -f "doc/man1/$_.pod" ) {
+        if ( ! -f "doc/man1/$_.pod"  ) {
```

Comment:
Why this?

Reply:
No reason. Oops. Or to see if I have your attention. Your pick :wink: